e2e/cypress/integration/ui/infra-proxy/chef-servers.spec.ts

```diff
@@ -0,0 +1,127 @@
+describe('chef server', () => {
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'infra';
+  const serverName = `${cypressPrefix} server ${now}`;
+  const generatedServerID = serverName.split(' ').join('-');
+  const customServerID = `${cypressPrefix}-custom-id-${now}`;
+  const serverFQDN = 'chef-server-1617089723092818000.com';
+  const serverIP = '176.119.50.159';
+
+  before(() => {
+    cy.adminLogin('/infrastructure/chef-servers').then(() => {
+      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      cy.get('app-welcome-modal').invoke('hide');
```

Comment:
You should be able to delete this line--it is redundant because it is already handled within the `cy.adminLogin` command. (Probably where you copied this from had been done earlier than that change.)